src/Microsoft.PowerShell.Commands.Management/commands/management/GetContentCommand.cs

```diff
@@ -338,13 +338,9 @@ private bool ScanForwardsForTail(ContentHolder holder, CmdletProviderContext cur
                 if (ReadCount <= 0 || (ReadCount >= tailResultQueue.Count && ReadCount != 1))
                 {
                     count = tailResultQueue.Count;
-                    ArrayList outputList = new ArrayList();
-                    while (tailResultQueue.Count > 0)
-                    {
-                        outputList.Add(tailResultQueue.Dequeue());
-                    }
+
                     // Write out the content as an array of objects
-                    WriteContentObject(outputList.ToArray(), count, holder.PathInfo, currentContext);
+                    WriteContentObject(tailResultQueue.ToArray(), count, holder.PathInfo, currentContext);
```

Comment:
OK, the original code clears `tailResultQueue` and the new code does not, but it does not matter because `tailResultQueue` is not used afterwards.